### PR TITLE
use HHS level data from AUS QLD

### PR DESCRIPTION
**Pull request by camjc**
AUS QLD have added a table, woohoo! Now they're reporting total confirmed to date instead of just new additions.

TODO
- [ ] Fix the total for QLD, it goes to 794 for some reason (double the total). Not sure why its doing this, hopefully someone can enlighten me.
- [ ] Figure out what is supposed to happen for "overseas" as a reporting area 😬 .
- [ ] How to map these HHS regions onto the geo map

### CODE REVIEW

File: src/events/crawler/scrapers/AUS/QLD/index.js [hunk 1]
```diff
@@ -16,18 +24,35 @@ const scraper = {
...
+      const HHSs = []; // See https://www.health.qld.gov.au/maps
+      const $ = await getCurrentArticlePage(this.url);
+      const $table = $('#content table');
+      const $trs = $table.find('tbody > tr:not(:last-child)');
+      $trs.each((index, tr) => {
+        const $tr = $(tr);
+        const cases = parse.number($tr.find('td:last-child').text());
+        const county = parse.string($tr.find('td:first-child').text());
+        HHSs.push({
+          county,
```
Comment (lazd) [inline src/events/crawler/scrapers/AUS/QLD/index.js]: Hmm, the previous scraper returned states, now this one has counties? Trying to wrap my head around the geography.

Comment (camjc) [reply src/events/crawler/scrapers/AUS/QLD/index.js]: Previous scraper was a state. Now it has subdivided areas called HHS. Can see the map in the comment.
Potentially I’d marked it wrong either before or after, but the idea is going from one state (Queensland) to a dozen or so sub-areas (which I figure aligns to USA counties).

Annoyingly one of the sub areas is “overseas” so not sure how to handle that either.

**Comment by lazd** [comment]
@camjc the total is double because it counts summed data and individual regions data

**Comment by camjc** [comment]
@lazd so this is intended behaviour or I should fix something?

**Comment by lazd** [comment]
@camjc intended behavior -- those totals are double counted.

**Comment by lazd** [comment]
@camjc you're seeing that in the build log output, not data, right?

**Pull request merged**